app/src/main/java/com/openclassrooms/realestatemanager/repository/SearchRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.openclassrooms.realestatemanager.repository
+
+import androidx.lifecycle.MutableLiveData
+import com.openclassrooms.realestatemanager.model.SearchCriteria
+
+class SearchRepository {
+
+    private val searchCriteriaLiveData: MutableLiveData<List<SearchCriteria>> = MutableLiveData()
+
+    fun getSearchCriteriaLiveData() = searchCriteriaLiveData
```

Comment:
```suggestion
    fun getSearchCriteriaLiveData() = searchCriteriaLiveData as LiveData<List<SearchCriteria>>
```

Sinon tu l'exposes en tant que mutable

Reply:
OK